src/solvers.jl

```diff
@@ -573,17 +573,18 @@ function vartypes_without_fixed(m::Model)
     return colCats
 end
 
-function collect_expr!(m, tmprow, terms::AffExpr)
+function collect_expr!(m, tmprow, terms::AffExpr, checkowned=true)
```

Comment:
Could you document what is supposed to happen when ``checkowned=false``? Why is it useful to have an option to ignore terms in the ``AffExpr``?